### PR TITLE
fix: regression on stacking layout

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/8174914128), [Test report](https://leather-wallet.github.io/playwright-reports/fix/locked-spacing)

Regression from past fix of https://github.com/leather-wallet/extension/pull/4857

Something changed causing the layout to be broken onto two lines.

### CODE REVIEW

**Comment by markmhendrickson** [comment]
Can we put a (visual) regression test in place for this?

**Comment by kyranjamie** [comment]
> Can we put a (visual) regression test in place for this?

Yes ser.

Had to do a bit of a refactor as the responsibility of displaying the information, and fetching the data were part of the same component. I've broken these, up, and written a storybook.

image

**Review by pete-watters — APPROVED** [review]
👍 code LGTM, just a typecheck to fix

File: src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.layout.tsx [hunk 1]
```diff
@@ -57,10 +58,14 @@ export function CryptoCurrencyAssetItemLayout({
         }
         captionRight={
           !rightElement && (
-            <>
-              <Caption>{balance.amount.toNumber() > 0 && address ? usdBalance : null}</Caption>
-              {additionalUsdBalanceInfo}
-            </>
+            <Caption>
```
Comment (fbwoolf) [inline src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.layout.tsx:61]: Thanks for fixing, this was prob me breaking it implementing the `ItemInteractive`. I remember encountering this component again recently, questioning if it is really a `layout` component now when it is just rendering an `ItemInteractive` that that wraps the `ItemLayout`? Maybe we don't even need to have the props passed to render here?

File: src/app/features/asset-list/components/stacks-balance-list-item.layout.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { TooltipProvider } from '@radix-ui/react-tooltip';
+import { Meta, StoryObj } from '@storybook/react';
+import BigNumber from 'bignumber.js';
+
+import { StacksBalanceListItemLayout } from './stacks-balance-list-item.layout';
+
+const meta: Meta<typeof StacksBalanceListItemLayout> = {
+  component: StacksBalanceListItemLayout,
+  tags: ['autodocs'],
+  title: 'Feature/StacksBalanceListItem',
```
Comment (fbwoolf) [inline src/app/features/asset-list/components/stacks-balance-list-item.layout.stories.tsx:10]: We should make sure to coordinate with @pete-watters on the new sections being added to our storybook. I noticed in his he has new sections too.

**Review by fbwoolf — APPROVED** [review]
Cool to see a storybook for a feature. 👍

File: src/app/features/asset-list/components/stacks-balance-list-item.layout.stories.tsx (hunk 2 above)
Comment (pete-watters) [reply src/app/features/asset-list/components/stacks-balance-list-item.layout.stories.tsx:10]: I actually decided to revert my changes to fit into the current system. Once everything is merged lets review it together. 

I had just added `Design System/` to the start of the path

**Pull request merged**